src/aria/jsunit/JawsTestCase.js

```diff
@@ -133,7 +133,9 @@ module.exports = Aria.classDefinition({
         assertJawsHistoryEquals : function (expectedOutput, callback, filterFn) {
             this.retrieveJawsHistory({
                 fn: function (response) {
+                    var globalFilterRegExp = /(Aria Templates tests frame|Aria Templates tests document)\n/gi;
 
+                    response = response.replace(globalFilterRegExp, "");
```

Comment:
Maybe I would put this filter in `retrieveJawsHistory` instead, as it is done for `"New tab page"`.